File: Programación/Códigos Java/Java/Unidad_2/src/funciones/Ejemplo1.java
package funciones;

import java.util.Scanner;

public class Ejemplo1 {
	public static void main(String[] args) {
		Ejemplo1 f = new Ejemplo1();

		int resultado = f.sumaNum(4, 7);
		System.out.println(resultado);
		
		String [] lista = f.generaLista5n();
		
		for(int i=0; i<5; i++) {
		System.out.println("Nombre nº"+i+":"+lista[i]);
		}

	}

	int sumaNum(int num1, int num2) {

		return num1 + num2;
	}

	String[] generaLista5n() {
		Scanner input = new Scanner(System.in);
		String[] Lista = {"", "", "", "", ""};
		
		for (int i = 0; i < 5; i++) {
			System.out.print("Introduce un Nombre: ");
			String nom = input.nextLine();
			Lista[i] = nom;
		}

		input.close();
		return Lista;
	}
}

File: Lenguaje de Marcas/Código VisualStudio/Programación/Python/Clase/Ejemplos/Ejemplos_Funciones/Ejercicios_Ejemplo/Ejemplo1.py
def exponente(num1, num2):
    if num2 == 0:
        resultado = 1

    for i in range(1, num2):
        resultado = resultado * num1
    return resultado

resultado_Fun = exponente(5,1)
print(resultado_Fun)

resultado_Fun = exponente(5,2)
print(resultado_Fun)

resultado_Fun = exponente(5,3)
print(resultado_Fun)